src/theme.ts

```diff
@@ -1,58 +1,56 @@
-const palette = {
-  red: "#ca2026",
-  danger: "#c2002f",
-  darkRed: "#c22032",
-  lightRed: "#e298a0",
-  paleRed: "#FBE7EA",
-  green: "#77af42",
-  lightGreen: "#8bc753",
-  paleGreen: "#e0edd3",
-  darkGreen: "#63a524",
-  darkerGreen: "4e7226",
-  paleYellow: "#ffffbb",
-  teal: "#0dc0de",
-  blue: "#007da4",
-  mediumBlue: "#026AA1",
-  lightBlue: "#34bdd8",
-  neutralLightBlue: "#0dc0dc",
-  tangerine: "#ffbd3e",
-  gray: "#5e5e5e",
-  darkGray: "#757575",
-  pale: "#d5d5d5",
-  light: "#e4e4e4",
-  white: "#ffffff",
-  neutralLightest: "#f9f9f9", // nearly white
-  neutralCool: "#f6f7f8", // cool bright gray
-  neutralBright: "#f5f5f5", // bright gray
-  neutralLighter: "#f1f1f1", // light gray
-  neutralLight: "#e5e5e5", // light gray
-  neutralMedium: "#a0a0a0", // light gray
-  neutral: "#818181", // gray
-  neutralThin: "#6f6f6f", // medium gray
-  neutralDark: "#5f6163", // dark gray
-  neutralFeedback: "#555", // another dark gray
-  neutralDarker: "#424242", // very dark gray
-  black: "#000000",
-  orange: "#D4450C"
-} as const;
+import { css, FlattenSimpleInterpolation } from "styled-components";
+import { buttonVariants } from "./theme/buttons";
+import { palette } from "./theme/palette";
 
 export const colors = {
   palette: palette,
+  button: {...buttonVariants, default: buttonVariants.orange},
```

Comment:
I don't understand why this theme file needs to know about buttons at all

Reply:
Maybe I'm making a mistake with the naming — that object was just gonna hold color variations, and with this you can just import `colors` and get access to everything in one clear namespace. But I think I see where you are going, consumers won't care about button colors directly so it could be confusing to expose this to them?

Reply:
Well if you renamed it to something like `colorSets` or something I would be less offended by putting them straight in the theme. I think rex has a similar construct but I forget what it's called

Reply:
Sure thing, I know I thought about this initially and wondered if "color" was awkward because I wanted to name things more like "light", "primary", "secondary" etc. (meant to rename orange and gray to those) but I should have spent more time considering it, thanks for the feedback I will update it to how rex does things

Reply:
Not sure if we need some naming tweaks but maybe this is closer to what we're looking for https://github.com/openstax/ui-components/pull/3/commits/4a70a4e40a472d420ea46d381fb21f594c5b4c2d